### PR TITLE
7837-Update API Docs

### LINKED ISSUE #7837: APIDoc value types should be capitalized
Many of the params in the APIDoc comments show types with all lowercase letters, like `{string}`, when they should be `{String}`. `{uuid}` should be stylized as `{UUID}`.

### PR DESCRIPTION
**Pull request by audiolion**
Fixes #7837 
### Changes
- Updated API Docs for api-v3
- Updated API Docs for top-level

---

UUID: 236e74bc-986d-41ad-9fa1-d4842ad72334

### CODE REVIEW

File: website/server/controllers/api-v3/tasks.js [hunk 1]
```diff
@@ -379,12 +379,12 @@ function _generateWebhookTaskData (task, direction, delta, stats, user) {
...
- * @apiParam {string} taskId The task _id or alias
- * @apiParam {string="up","down"} direction The direction for scoring the task
+ * @apiParam {String} taskId The task _id or alias
+ * @apiParam {String="up","down"} direction The direction for scoring the task
  *
- * @apiSuccess {object} data._tmp If an item was dropped it'll be returned in te _tmp object
- * @apiSuccess {number} data.delta
- * @apiSuccess {object} data The user stats
+ * @apiSuccess {Object} data._tmp If an item was dropped it'll be returned in te _tmp object
+ * @apiSuccess {Number} data.delta The delta
```
Comment (audiolion) [inline website/server/controllers/api-v3/tasks.js:386]: if anyone has a better idea about a description for this let me know

File: website/server/controllers/api-v3/tags.js [hunk 2]
```diff
@@ -119,10 +119,10 @@ api.updateTag = {
  * @apiName ReorderTags
  * @apiGroup Tag
  *
- * @apiParam {tagId} UUID Id of the tag to move
- * @apiParam {to} number Position the tag is moving to
+ * @apiParam {UUID} tagId Id of the tag to move
+ * @apiParam {Number} number Position the tag is moving to
```
Comment (crookedneighbor) [inline website/server/controllers/api-v3/tags.js]: I _think_ the param is supposed to be called to

File: website/server/controllers/api-v3/status.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ let api = {};
  * @apiName GetStatus
  * @apiGroup Status
  *
- * @apiSuccess {status} data.status 'up' if everything is ok
+ * @apiSuccess {Status} data.status 'up' if everything is ok
```
Comment (crookedneighbor) [inline website/server/controllers/api-v3/status.js]: This is a String

File: website/server/controllers/api-v3/tags.js [hunk 4]
```diff
@@ -120,7 +120,7 @@ api.updateTag = {
  * @apiGroup Tag
  *
  * @apiParam {UUID} tagId Id of the tag to move
- * @apiParam {Number} number Position the tag is moving to
+ * @apiParam {to} number Position the tag is moving to
```
Comment (crookedneighbor) [inline website/server/controllers/api-v3/tags.js]: Sorry, wasn't clear. I think it is: `{Number} to`

Comment (audiolion) [inline website/server/controllers/api-v3/tags.js]: ohhh I was like, ok this is weird but I trust you 😛

**Pull request merged**